.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-name: Build
+name: tuild
```

Comment:
What?

Reply:
Fixed yet :D I revoke this commit is miss take i guess :D